### DISCUSSION TITLE
ADM5 Klipper Mod v00.01-alpha

### DISCUSSION THREAD

**Discussion by xblax** [xblax/flashforge_ad5m_klipper_mod#1] [Announcements]
First alpha release for closed user group testing. Development testing was done on a regular 5M, but should work on the 5M Pro as well.

Note: For 5M Pro, please rename the archive to `Adventurer5MPro-*.tgz`, otherwise it's not detected by the installer.

<hr /><em>This discussion was created from the release <a href='https://github.com/xblax/flashforge_adm5_klipper_mod/releases/tag/v00.01-alpha'>ADM5 Klipper Mod v00.01-alpha</a>.</em>

**Comment by nitronov** [comment]
I still don’t understand how the auto-delete script works. Just need to rename the firmware file to klipper_mod_remove?

**Reply by jlodew0** [reply]
you just need a file on your usb stick with a file that's named klipper_mod_remove. It can be a empty file as well.

**Reply by nitronov** [reply]
> you just need a file on your usb stick with a file that's named klipper_mod_remove. It can be a empty file as well.

txt or anyone?

**Reply by jlodew0** [reply]
without file extension.

You can check this script to understand how it works:
https://github.com/xblax/flashforge_adm5_klipper_mod/blob/master/build_overlay/etc/init/S90klipper_mod
